src/Bridge/Symfony/Bundle/Test/Constraint/MatchesJsonSchema.php

```diff
@@ -80,4 +77,20 @@ protected function additionalFailureDescription($other): string
 
         return implode("\n", $errors);
     }
+
+    private function canonicalizeJson($data)
+    {
+        if (!(\is_array($data) || \is_object($data))) {
```

Comment:
```suggestion
        if (!\is_array($data) && !\is_object($data)) {
```

For readability?

Reply:
Okay... Though to me `NOT (A OR B)` is clearer than `NOT A AND NOT B`.